src/requests/request-helpers.ts

```diff
@@ -177,3 +178,40 @@ export function formatEmbedContentInput(
   }
   return params;
 }
+
+export function formatGenerateImageInput(
+  params: ImageGenerationRequest | string,
+): ImageGenerationRequest {
+  let formattedRequest: ImageGenerationRequest;
+  if (typeof params === "string") {
+    formattedRequest = { prompt: params };
+  } else {
+    formattedRequest = params as ImageGenerationRequest;
+  }
+  return formattedRequest;
+}
+
+export function convertFromImageGenerationRequest(
+  model: string,
+  request: ImageGenerationRequest,
+): PredictRequest {
+  const instances = [{ prompt: request.prompt }];
+  const sampleImageSize = Math.max(request.width || 0, request.height || 0);
+  const parameters = {
+    negativePrompt: request.negativePrompt,
```

Comment:
Can you start with `...request` for all the request parameters that don't need any modification and then add the modified ones after?